### ISSUE TITLE
BUG - Problem 25 - Base Code Layout

### ISSUE THREAD

**Issue by Selbl** [Open-Deep-ML/DML-OpenProblem#327]
The base code for problem 25 is malformed, the import numpy as np and function definitions are jumbled together like this:

`import numpy as npdef train_neuron(features: np.ndarray, labels: np.ndarray, initial_weights: np.ndarray, initial_bias: float, learning_rate: float, epochs: int) -> (np.ndarray, float, list[float]):`

You can fix this by adding a linebreak after the import numpy as np line.

**Comment by Open-Deep-ML** [comment]
thank you for letting us know, it is now fixed (should take about 30 min to update on the site)
